.circleci/config.yml

```diff
@@ -153,8 +149,14 @@ workflows:
   build-test-and-approval-deploy:
     jobs:
       - danger
-      - flutter_tests
-      - flutter_tests_2-10-5
+      - test_flutter:
+          name: test_flutter-stable
```

Comment:
I think we can use [CircleCI's matrix jobs](https://circleci.com/docs/using-matrix-jobs/) approach instead of separating each one that way.